src/include/miopen/find_solution.hpp

```diff
@@ -77,36 +77,37 @@ auto FindSolutionImpl(rank<1>,
         {
             using PerformanceConfig = decltype(s.GetDefaultPerformanceConfig(context, problem));
             PerformanceConfig config{};
-            if(db.Load(problem, s.SolverDbId(), config))
```

Comment:
We want the passed in string perf config to take precedence over the one in the data base. If the entry exists in the database then the passed in config will not be used, which defeats the purpose of the function argument.